src/node.js

```diff
@@ -22,6 +22,50 @@
 
     EventEmitter.call(process);
 
+    process.on('warning', (warning) => {
+      if (process.noProcessWarnings) return;
+      if (process.traceProcessWarnings)
+        console.error(warning.stack);
+      else
+        console.error(`${warning.name}: ${warning.message}`);
+    });
+    process.on('deprecation', (warning) => {
+      if (process.noDeprecation) return;
+      if (process.traceDeprecation)
+        console.error(warning.stack);
+      else
+        console.error(`${warning.name}: ${warning.message}`);
```

Comment:
Can this line be changed in some way to allow the underlying object choose how it wants to be stringified?